packages/flutter_tools/test/commands/devices_test.dart

```diff
@@ -20,6 +20,7 @@ void main() {
   group('devices', () {
     setUpAll(() {
       Cache.disableLocking();
+      debugDisableWebAndDesktop = true;
```

Comment:
Add a TODO here? It doesn't seem like we want no testing of web and desktop devices in the long term.